src/main/resources/avro/bdg.avdl

```diff
@@ -252,8 +257,39 @@ record AlignmentRecord {
   union { null, Contig } mateContig = null;
 }
 
+record Sequence {
+  union { null, string } bases = null;
+  union { null, string } qualities = null;
+}
+
 /**
- Enumeration for DNA/RNA bases. For codes outside of ACTGU, see the IUPAC 
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
+*/
+record Fragment {
+  /**
+   The name of this Fragment.
+   */
+  union { null, string } readName = null;
```

Comment:
This comment should probably go on #44; I pulled in the commits from there by accident.